media-libs/libavif/libavif-0.8.0.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="Library for encoding and decoding .avif files"
+DESCRIPTION="This library aims to be a friendly, portable C implementation of \
+the AV1 Image File Format, as described here: \
+ \
+https://aomediacodec.github.io/av1-avif/ \
+ \
+It is a work-in-progress, but can already encode and decode all AOM supported \
+YUV formats and bit depths (with alpha). \
+ \
+For now, it is recommended that you checkout/use tagged releases instead of \
+just using the master branch. I will regularly create new versions as \
+bugfixes and features are added."
+HOMEPAGE="https://https://github.com/AOMediaCodec/libavif"
+COPYRIGHT="2019 Joe Drago"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/AOMediaCodec/libavif/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="c917f4a27add61b8ad15f328090942d519460b71219f31e60e5fe6aa8cf3dc4e"
+
+ARCHITECTURES="?x86_gcc2 ?x86 !x86_64 ?arm ?riscv64 ?sparc ?m68k"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="5.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libavif$secondaryArchSuffix = $portVersion
+	lib:libavif$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libdav1d$secondaryArchSuffix
+	"
+
+PROVIDES_tools="
+	cmd:avifenc
+	cmd:avifdec
```

Comment:
On gcc2h they are "cmd:avifen_x86" so:
```diff
-	cmd:avifenc
-	cmd:avifdec
+	cmd:avifenc$secondaryArchSuffix
+	cmd:avifdec$secondaryArchSuffix
```
Or maybe use -DCMAKE_INSTALL_BINDIR=$prefix/bin ? (not tested)

Reply:
I’m not sure I understand your second suggestion, I did add the `$secondaryArchSuffix` but from what I understand the `packageEntries` command already does the correct work even on `$binDir`.

Reply:
Maybe this will explain a bit better on what I'm trying to say here: https://github.com/haikuports/haikuports/blob/43a37f4c039afa53b47f4ab79279a876d476ce6a/media-libs/partio/partio-1.1.0.recipe#L25

Reply:
> I’m not sure I understand your second suggestion, I did add the `$secondaryArchSuffix` but from what I understand the `packageEntries` command already does the correct work even on `$binDir`.

On x86_gcc2 $bindir = /prefix/bin/x86 (for secondaryArchSuffix)
On x86_x64 $bindir = /prefix/bin

If a package can't be build for primary architecture "!x86_gcc2" then we want the binaries installed in $prefix/bin rather then $prefix/bin/x86 for secondary architecture "x86" (secondaryArchSuffix), hence we use $commandBinDir and $commandSuffix then